CHANGELOG.md

```diff
@@ -257,4 +257,22 @@ All notable changes to this project will be documented in this file. The format
  - Fix: AKO is not updating the ingress status when annotation `passthrough.ako.vmware.com/enabled: "true"` is added to the ingress.
  - Fixes LoadBalancer service creation with named ports in NodePortLocal deployment.
  - Fix: Every SEGroup used in the AviInfraSetting is getting configured with the labels even when `disableStaticRouteSync` is set to `true`.
- - Fix: AKO pod keeps getting error "panic: runtime error: slice bounds out of range" then goes into CrashLoopBackOff state.
\ No newline at end of file
+ - Fix: AKO pod keeps getting error "panic: runtime error: slice bounds out of range" then goes into CrashLoopBackOff state.
+
+## AKO-1.8.1
+
+### Added
+ - AKO now claims support for Kubernetes 1.24.
+ - Support for `AviInfraSetting` CRD in Passthrough Ingresses/Routes.
+ - Support for ISTIO mTLS authentication. (Tech-preview)
+ - Support for IPv4/IPv6 dual-stack networking. (Tech-preview)
+
+### Changed
+ - Added a knob to block the processing of objects in system namespaces in OpenShift and K8s clusters.
```

Comment:
We can add this as part of `Added` section.

Reply:
I have addressed the comment.